src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -294,28 +294,6 @@ void hiopVectorRaja<MEM, POL>::copyFrom(const double* local_array)
   }
 }
 
-/// @brief Copy from vec the elements specified by the indices in index_in_src.
-template<class MEM, class POL> 
-void hiopVectorRaja<MEM, POL>::copy_from(const hiopVector& vec, const hiopVectorInt& index_in_src)
```

Comment:
I am wondering why the implementation of two `copy_from` methods was removed...

Reply:
They are identical to [copy_from_indexes(const double* vv, const hiopVectorInt& index_in_src)
](https://github.com/LLNL/hiop/blob/b7d5205212424e8a794ae950c6da5f05f5d90cd0/src/LinAlg/hiopVectorRajaImpl.hpp#L343-L344)
and 
[copy_from_indexes(const hiopVector& vv, const hiopVectorInt& index_in_src)](https://github.com/LLNL/hiop/blob/b7d5205212424e8a794ae950c6da5f05f5d90cd0/src/LinAlg/hiopVectorRajaImpl.hpp#L320-L321)